packages/core/directive.js

```diff
@@ -0,0 +1 @@
+export { directive, Directive } from 'lit-html/directive.js';
```

Comment:
think this points to https://github.com/lit/lit/blob/lit-html-1.x/src/lit-html.ts#L36 and could be exported from index.js as well?